.dir-locals.el

```diff
@@ -0,0 +1 @@
+((nil . ((cider-clojure-cli-global-options . "-A:dev:test-clj"))))
```

Comment:
this file was in .gitignore, but imho at least this config is useful to have for everyone.
I can remove it though if people prefer to have their own .dir-locals outside of git.